sdk/identity/azure-identity-extensions/Azure-Database-for-PostgreSQL-README.md

```diff
@@ -299,5 +299,5 @@ the [contributing guide](https://github.com/Azure/azure-sdk-for-java/blob/main/C
 [docs]: https://azure.github.io/azure-sdk-for-java/
 [postgresql_product_docs]: https://docs.microsoft.com/azure/postgresql/single-server/overview
```

Comment:
```suggestion
[postgresql_product_docs]: https://learn.microsoft.com/azure/postgresql/single-server/overview
```

Reply:
it won't let me leave comments on lines that weren't near a modification but there are others here.